README.md

```diff
@@ -25,13 +25,14 @@ global manipulation. Our goal with **lab** is to keep the execution engine as si
 - `-C`, `--colors` - enables or disables color output. Defaults to console capabilities.
 - `-d`, `--dry` - dry run. Skips all tests. Use with `-v` to generate a test catalog. Defaults to `false`.
 - `-D`, `--debug` - print the stack during a domain error event.
+- `--debug-brk` - break on the first line of application code.
```

Comment:
It's passed to node so it is the same. I don't see any code intercepting the 1st test call.